Rework crontab usage (5.0) (#2664)
ezsystems/developer-documentation: docs/discounts/configure_discounts.md

php bin/console messenger:consume ibexa.messenger.transport --bus=ibexa.messenger.bus
```

> 3\. Periodically run the following command, about one time a day or more often:

Reviewer: How about "Run the following command periodically, at least once a day:"?

Author: ```suggestion
3\. Run the following command periodically, at least once a day:
```

``` bash
php bin/console ibexa:discounts:reindex
```

> For more information about scheduling a command, see [Additional scheduled tasks and advanced usage](install_ibexa_dxp.md#additional-scheduled-tasks-and-advanced-usage).

Reviewer: ```suggestion
For more information about command scheduling, see [Additional scheduled tasks and advanced usage](install_ibexa_dxp.md#additional-scheduled-tasks-and-advanced-usage).
```
